Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -281,28 +282,32 @@ public async Task ReconnectTurn(int listenPort, CancellationToken cancellationTo
             return !_control.Client.LocalEndPoint.Equals(mapped);
         }
 
-        public async Task<bool> IsConnectable(
-            CancellationToken cancellationToken = default)
+        public async Task<bool> IsConnectable(int probeLength = ConnectableProbeLength)
         {
+            CancellationTokenSource cts = new CancellationTokenSource();
```

Comment:
```suggestion
            using var cts = new CancellationTokenSource();
```

we can omit `finally` with `using` IMHO 😄